start-from-zero/Javascript/src/views/new-evaluation/new-evaluation.js

```diff
@@ -0,0 +1,725 @@
+    const selectArr = [];
+    options.sel.forEach(function(n){
+        selectArr.push(`
+            <option value="${n.value}">${n.language}</option>
+            `);
+    });
+    return selectArr.join('');
+}
+
+// const SelectName = function(options={}){
+//     const selectNameArr = [];
+//     options.des.forEach(function(n){
+//         selectNameArr.push(`
+//         <select name="${n.selectDescription}" class="select">
+//             ${SelectLevel()}
+//         </select> 
+//         `);
+//     });
+//     return selectNameArr.join('');
+// }
+
+
+
+const NewEvaluationTable = function(options = {}){
+    return `
+<section class="form-candidate">
+    <div class="container">
+       <div class="row">
+            <div class="col-md-12">
+                <form action="" class="user-input" method='post'>
+                <div class="field-ev">
+                    <input class='input-username' type="text" name='username' placeholder="Username">
+                    <input class='input-password' type="password" name='password' placeholder="Password">
+                    <input class='input-calendar' type="date">
+                </div>
+
+
+                <h4 class='technical-header'>Technical Level</h4>
+                <table class='technical-level-table'>
+
+                    <tbody class='tbody-new'>
+                        <tr class='technical-level'>
+                            ${TechnicalLevel({
+                                el: options.el
+                            })}
+                        </tr>
+
+
+                        <tr class="technical-level-category">
+                            <td>
+                                <input class='input-checkbox' type="checkbox" name='trainee'>
+                            </td>
+                            <td>
+                                <input class='input-checkbox' type="checkbox" name='junior1'>
+                                <input class='input-checkbox' type="checkbox" name='junior2'>
+                                <input class='input-checkbox' type="checkbox" name='junior3'>
+                            </td>
+                            <td>
+                                <input class='input-checkbox' type="checkbox" name='middle1'>
+                                <input class='input-checkbox' type="checkbox" name='middle2'>
+                                <input class='input-checkbox' type="checkbox" name='middle3'>
+                            </td>
+                            <td>
+                                <input class='input-checkbox' type="checkbox" name='senior1'>
+                                <input class='input-checkbox' type="checkbox" name='senior2'>
+                                <input class='input-checkbox' type="checkbox" name='senior3'>
+                            </td>
+                        </tr>
+                    </tbody>
+                </table>
+
+
+
+
+                <div class="textarea-wrap">
```

Comment:
create a textarea component for each <div class="textarea-wrap">

Reply:
Solved